clang/lib/AST/ASTImporter.cpp

```diff
@@ -735,6 +740,38 @@ namespace clang {
     // that type is declared inside the body of the function.
     // E.g. auto f() { struct X{}; return X(); }
     bool hasReturnTypeDeclaredInside(FunctionDecl *D);
+    
+    Expected<ConstraintSatisfaction> FillConstraintSatisfaction(const ASTConstraintSatisfaction& from) {
```

Comment:
Variable names should start with uppercase letters. There should be always a space after commas (but clang-format should add it?).